test/sanitizer_suppressions/ubsan

```diff
@@ -12,6 +12,7 @@ float-divide-by-zero:wallet/wallet.cpp
 # contains files in which we expect unsigned integer overflows to occur. The
 # list is used to suppress -fsanitize=integer warnings when running our CI UBSan
 # job.
+unsigned-integer-overflow:*/include/c++/*/bits/basic_string.tcc
```

Comment:
This will hide other unsigned int overflows that occur from different call sites. I think it should eventually be deleted, no?

Reply:
AFAICT we can't do anything meaningful about these unsigned integer wraparounds in `basic_string.tcc`. I think that suppressing them is fine: unsigned integer wraparound is not UB after all :)